.cirrus.yml

```diff
@@ -124,7 +124,7 @@ vendor_task:
 
     # Runs within Cirrus's "community cluster"
     container:
-        image: docker.io/library/golang:1.24.0
+        image: docker.io/library/golang:1.24.7
```

Comment:
would it not be better to just use the `1.24` tag without patch release instead? That way it should always use the latest which means we don't have to update this manually so often.

Reply:
This value exactly matches what's in go.mod.

Reply:
right but we get bumps like https://github.com/containers/buildah/pull/6385 where dependencies like to bump to go 1.24.2 for whatever reasons... so then it means it requires a manual update here.

Reply:
The last several PRs that renovate opened didn't need such a change added to them.  If the bot had done it in #6385, I don't know that we'd even be talking about it.

Reply:
Well the change is added when said dependency declares such go version. So if renovate would open the go.podman.io/common bump for example it would run into the exact same issue.

Reply:
I will rebase this PR with version changed to `1.24` once @nalind agrees.

Reply:
I thought renovate couldn't handle the layout of the container-libs repository (it hasn't opened one for any of the components we get from it), and keeping the dependency updates automated was part of the motivation for podmanbot.
The do-not-merge bit isn't only being set because it's still being debugged?